src/libstd/process.rs

```diff
@@ -118,9 +122,11 @@ impl Read for ChildStderr {
 /// ```
 /// use std::process::Command;
 ///
-/// let output = Command::new("sh").arg("-c").arg("echo hello").output().unwrap_or_else(|e| {
-///   panic!("failed to execute process: {}", e)
-/// });
+/// let output = Command::new("sh")
+///     .arg("-c")
```

Comment:
I think our current style for this form of indentation is to line up the `.` of `.arg` with one of the `:` characters in `Command` (which one depends who you ask!)